styleguide/index.html

```diff
@@ -0,0 +1 @@
+<!DOCTYPE html><html><head><meta charset="UTF-8"><meta name="viewport" content="width=device-width, initial-scale=1.0"><meta http-equiv="X-UA-Compatible" content="ie=edge"><title>Volto Style Guide</title></head><body><div id="rsg-root"></div><script src="build/bundle.0ac92aab.js"></script></body></html>
```

Comment:
Where this come from?

@ajayns @nileshgulia1 ?

Reply:
Sorry. It did came along with styleguidist. I'll remove it. :+1: